charts/bulk-scan-payment-processor/values.yaml

```diff
@@ -3,3 +3,10 @@ java:
   ingressHost: bulk-scan-payment-processor-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   environment:
   image: 'https://hmctspublic.azurecr.io/bulk-scan/payment-processor:latest'
+  keyVaults:
+    "bulk-scan":
+      resourceGroup: payment-processor
+      secrets:
+        - site-id-probate
+        - site-id-divorce
+        - site-id-finrem
```

Comment:
you will have to create those secrets manually

Reply:
ok